src/js/chatService.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-alert */
```

Comment:
I think this got left in my mistake. I don't see `alert` being used in this module.